config/routes.rb

```diff
@@ -279,6 +279,16 @@
       resource :mentorship, only: %i[new create] do
         get :confirmation, on: :collection
       end
+
+      namespace :change_name_wizard, path: 'change-name', controller: '/schools/change_name_wizard' do
+        get 'edit', action: :new
+        post 'edit', action: :create
+
+        get 'check-answers', action: :new
+        post 'check-answers', action: :create
+
+        get 'confirmation', action: :new
```

Comment:
I'd gone with `:edit`, :confirm`, and `:success`.

But I think this makes more sense, especially since "Check answers" is a concept/pattern we're all familiar with 👌🏼